packages/xray-toolset/PKGBUILD

```diff
@@ -0,0 +1,60 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=xray-toolset
+pkgver=EvilPot.0.0.2.r1.gaee9f9c
+pkgrel=1
+pkgdesc='A complete security assessment tool that supports common web security issues scanning and custom POC.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-webapp')
+url='https://github.com/chaitin/xray'
+license=('custom:unknown')
+depends=()
+makedepends=('git')
+source=("git+https://github.com/chaitin/xray.git"
+        "git+https://github.com/chaitin/xray-plugins.git"
+        "https://github.com/chaitin/xray/releases/download/xpoc-0.1.0/xpoc_linux_amd64"
```

Comment:
add xpoc as dependency and built it https://github.com/chaitin/xpoc

Reply:
The `xpoc` repository has not source code to build the binary. Should I create a dedicated package for each one of them and still use the pre-build binaries?

Reply:
True. Then if it's closed-source, we would avoid shipping pre-built binaries from untrusted sources.

Reply:
I agree. At this point I would close this PR. Can you please close https://github.com/BlackArch/blackarch/issues/3310 by explicitly writing the reason you just mentioned?